src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -867,11 +868,21 @@ class AccessionStore(
                     1)
                 .else_(0))
 
+    val speciesQuery =
+        dslContext
+            .select(DSL.countDistinct(ACCESSIONS.SPECIES_ID))
+            .from(ACCESSIONS)
+            .innerJoin(SPECIES)
```

Comment:
Maybe replace `innerJoin` with `join` since that's the style we use elsewhere in the code (inner join is the default join type).